CMakeLists.txt

```diff
@@ -84,7 +84,8 @@ if(NOT KCONFIGLIB)
   $ pip install kconfiglib")
 endif()
 
-if(EXISTS ${BOARD_CONFIG} AND EXISTS ${BOARD_CONFIG}/defconfig)
+if(EXISTS ${NUTTX_DIR}/${BOARD_CONFIG}
```

Comment:
how about if BOARD_CONFIG is an absolute path?

Reply:
> how about if BOARD_CONFIG is an absolute path?

🙏Yes thanks, I made a mistake.
I submitted a new modification, please review.
since the configuration pair `<board>[/:]<config>` contains the character `/`, we cannot judge it through `IS_ABSOLUTE`, so I used `EXSIST` to judge both the absolute and relative paths.
and also use `NUTTX_BOARD_ABS_DIR` instead of `BOARD_CONFIG`.

Reply:
Please merge the  BOARD_CONFIG related commits into one.

Reply:
> Please merge the BOARD_CONFIG related commits into one.

Done, rebased